nirc_ehr/resources/schemas/dbscripts/postgresql/nirc_ehr-22.007-22.008.sql

```diff
@@ -0,0 +1,140 @@
+ALTER TABLE nirc_ehr.ProtocolUsage DROP COLUMN SegmentId;
+ALTER TABLE nirc_ehr.ProtocolUsage ADD COLUMN Species integer;
+
+ALTER TABLE nirc_ehr.AnimalReqOrder DROP COLUMN Segment;
+ALTER TABLE nirc_ehr.AnimalReqOrder ADD COLUMN Protocol integer;
+ALTER TABLE nirc_ehr.AnimalReqOrder ADD COLUMN Species integer;
+
+ALTER TABLE nirc_ehr.Account DROP COLUMN cage;
+ALTER TABLE nirc_ehr.Account DROP COLUMN room;
+ALTER TABLE nirc_ehr.Account DROP COLUMN building;
+
+CREATE TABLE nirc_ehr.ProtocolStress
```

Comment:
We generally have trouble with tables that don't have a PK. I suggest adding a RowId if there's not a natural PK in the data itself.

Reply:
Ok added RowId